make/autoconf/jdk-options.m4

```diff
@@ -496,9 +496,15 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_LEAK_SANITIZER],
 #
 AC_DEFUN_ONCE([JDKOPT_SETUP_UNDEFINED_BEHAVIOR_SANITIZER],
 [
+  UTIL_ARG_WITH(NAME: additional-ubsan-checks, TYPE: string,
+    DEFAULT: [],
+    DESC: [Custom ubsan checks],
+    OPTIONAL: true)
```

Comment:
Please use 4 spaces for continuation indent (see point 2 https://openjdk.org/groups/build/doc/code-conventions.html)